syscall/zsyscall_windows.go

```diff
@@ -18,7 +18,7 @@ var (
 )
 
 func GetModuleFileName(hmodule uint32, buf *uint16, buflen uint32) (n uint32, err error) {
-	r0, _, e1 := syscall.Syscall(procGetModuleFileNameW.Addr(), 3, uintptr(hmodule), uintptr(unsafe.Pointer(buf)), uintptr(buflen))
+	r0, _, e1 := syscall.Syscall(procGetModuleFileNameW.Addr(), 3, uintptr(hmodule), uintptr(unsafe.Pointer(buf)), uintptr(buflen)) // #nosec
```

Comment:
I can add it to the `-skip` list. Should I just do this file, or all of `syscall/`?